casper/src/test/scala/coop/rchain/casper/batch1/MultiParentCasperMergeSpec.scala

```diff
@@ -46,39 +46,6 @@ class MultiParentCasperMergeSpec extends FlatSpec with Matchers with Inspectors
     }
   }
 
-  // TODO enable tests below
-//  it should "handle multi-parent blocks correctly when they operate on stdout" in effectTest {
-//    def echoContract(no: Int) =
-//      Rho(s"""new stdout(`rho:io:stdout`) in { stdout!("Contract $no") }""")
-//    merges(echoContract(1), echoContract(2), Rho("Nil"))
-//  }
-//
-//  it should "not conflict on registry lookups" in effectTest {
```

Comment:
I think this one is still a worthy one. We can create another issue seperate for that.

Reply:
This can be extended to our base rholang library and see what would cause conflicts.

Reply:
https://github.com/rchain/rchain/issues/3381